app/models/like.rb

```diff
@@ -0,0 +1,16 @@
+class Like < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :posts
+
+  after_save :update_like_counter
+
+  private
+
+  def update_like_counter
+    if author.likes_counter
+      author.increment!(:likes_counter)
+    else
+      author.update(likes_counter: 1)
+    end
+  end
```

Comment:
- [ ] This method should be called when a user likes a single post. If you look at your `db/schema.rb` file, you will see that there is a column called `likes_counter` in the `Posts` table. This means that it is the likes_counter that will increase once a user likes a post. To reflect this, kindly make it `post.increment!(:likes_counter)` .👍